source/administration/config-settings.rst

```diff
@@ -3696,6 +3696,17 @@ Disable Bots When Owner Is Deactivated
 | This feature's ``config.json`` setting is ``"DisableBotsWhenOwnerIsDeactivated": true`` with options ``true`` and ``false``.                                         |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Latex
+^^^^^^^^^^^^^
+
+**True**: Enables rendering of latex code.
+
+**False**: Disables rendering of latex code to prevent the app from crashing when sharing code that might outgrow assigned memory.
```

Comment:
```suggestion
**False**: Disables rendering of latex code to prevent the app from crashing when sharing code that might outgrow assigned memory. When disabled, latex code will be highlighted.
```